sirepo/pkcli/job_cmd.py

```diff
@@ -241,6 +233,20 @@ def _do_sequential_result(msg, template):
     return r
 
 
+def _do_stateless_compute(msg, template):
+    try:
+        return template.stateless_compute(msg.data)
```

Comment:
One of the problems with getApplicationData is that there's no API for the sub functions. Solve the dispatch so that it calls something like stateless_compute_setup_command.

Reply:
17a94973c